cognitive/src/test/scala/com/microsoft/azure/synapse/ml/cognitive/split4/MultivariateAnamolyDetectionSuite.scala

```diff
@@ -198,3 +207,128 @@ class MultiAnomalyEstimatorSuite extends EstimatorFuzzing[MultivariateAnomalyEst
+      .transform(emptyDf)
+      .withColumn("status", col("result.summary.status"))
+      .withColumn("variableStates", col("result.summary.variableStates"))
+      .select("status", "variableStates")
+      .collect()
+
+    assert(result.head.getString(0).equals("READY"))
+    assert(result.head.getSeq(1).length.equals(3))
+
+    madDelete(model.getModelId)
+
+    smae.cleanUpIntermediateData()
+  }
+
+  test("SimpleMultiAnomalyEstimator basic usage with endpoint and sasToken") {
+
+    val smae = simpleMultiAnomalyEstimator.setSlidingWindow(200)
+      .setStorageName("anomalydetectiontest")
+      .setStorageKey(storageKey)
+      .setEndpoint("https://anomalydetectiontest.blob.core.windows.net/")
+      .setSASToken(storageSASToken)
+    val model = smae.fit(emptyDf)
+    val diagnosticsInfo = smae.getDiagnosticsInfo
+    assert(diagnosticsInfo.variableStates.get.length.equals(3))
+
+    val result = model
+      .setSource(smae.getSource).setStartTime(startTime).setEndTime(endTime)
+      .setOutputCol("result")
+      .transform(emptyDf)
+      .withColumn("status", col("result.summary.status"))
+      .withColumn("variableStates", col("result.summary.variableStates"))
+      .select("status", "variableStates")
+      .collect()
+
+    assert(result.head.getString(0).equals("READY"))
+    assert(result.head.getSeq(1).length.equals(3))
+
+    madDelete(model.getModelId)
+
+    smae.cleanUpIntermediateData()
+  }
+
+  test("Throw errors if alignMode is not set correctly") {
+    val caught = intercept[IllegalArgumentException] {
+      simpleMultiAnomalyEstimator.setSlidingWindow(200).setAlignMode("alignMode").fit(df)
+    }
+    assert(caught.getMessage.contains("parameter alignMode given invalid value"))
+  }
+
+  test("Throw errors if slidingWindow is not between 28 and 2880") {
+    val caught = intercept[IllegalArgumentException] {
+      simpleMultiAnomalyEstimator.setSlidingWindow(20).fit(df)
+    }
+    assert(caught.getMessage.contains("parameter slidingWindow given invalid value"))
+  }
+
+  test("Throw errors if required fields not set") {
+    val caught = intercept[AssertionError] {
+      new SimpleMultiAnomalyEstimator()
+        .setSubscriptionKey(anomalyKey)
+        .setLocation("westus2")
+        .setOutputCol("result")
+        .fit(df)
+    }
+    assert(caught.getMessage.contains("Missing required params"))
+    assert(caught.getMessage.contains("slidingWindow"))
+    assert(caught.getMessage.contains("startTime"))
+    assert(caught.getMessage.contains("endTime"))
+  }
+
+  override def testSerialization(): Unit = {
+    println("ignore the Serialization Fuzzing test because fitting process takes more than 3 minutes")
+  }
+
+  override def testExperiments(): Unit = {
+    println("ignore the Experiment Fuzzing test because fitting process takes more than 3 minutes")
+  }
+
+  override def afterAll(): Unit = {
+    val models = madListModels().parseJson.convertTo[MADListModelsResponse].models.map(_.modelId)
```

Comment:
this could mess up other concurrent running instances in the build if this cleans up while another instance is running. Consider keeping some hidden global variable to keep track of which specific models are created in this JVM or robustly clean up each individual one created.